### DISCUSSION TITLE
make-declaration-of-intent-for-sungbin-club/

### DISCUSSION THREAD

**Discussion by giscus** [devmeeple/devmeeple.github.io#18] [Comments]
# make-declaration-of-intent-for-sungbin-club/

무르익은 단풍 따라 워밍업 클럽 일정도 수료식만 남겨뒀다. 사실상 끝났다. 의존 없는, 스스로 완벽하게 제어 가능한 시간이다.
여러 실험 중 '성빈클럽'에 참여했다. 조금 다른 건 러너가 아닌 서포터로 참여한다. 다시 질문을 던진다. 그대들 어떻게 살 것인가 내가그린 성빈클럽 왜 성빈클럽인가요?  워밍업 클럽에서 느낀 점이 같았다. 스터디에 참여할 때 항…

https://devmeeple.github.io/make-declaration-of-intent-for-sungbin-club/

**Comment by SungbinYang** [comment]
태근님 서포터로 참여해주셔서 감사합니다. 태근님께도 좋은 영향을 주길 바라겠습니다!

**Reply by devmeeple** [reply]
의견 감사합니다. 
서로 상승곡선에 올라타길 바랍니다.
응원합니다.
